app/services/availability_service.py

```
@@ -437,18 +457,19 @@ async def check_availability(self, request: AvailabilityRequest) -> Availability
+                        available_count=1 if res.available != "unknown" else 0,
                         phone_number=room_detail.phoneNumber,
                         display_name=room_detail.displayName,
                         rooms=[room_resp]
                     )
-                    AvailabilityService._update_min_prices(branch_dict[bid], res.available, total_price)
+                    AvailabilityService._update_min_prices(branch_dict[bid], res.available, total_price, is_partial)
                 else:
                     branch_obj = branch_dict[bid]
                     branch_obj.rooms.append(room_resp)
-                    branch_obj.available_count += 1
+                    if res.available is not "unknown":
```

Ruff (0.15.2)

[error] Use `!=` to compare constant literals

Replace `is not` with `!=`

(F632)